src/gatling/simulations/uk/gov/hmcts/reform/amlib/performance/scenarios/BasicScenarios.scala

```diff
@@ -34,7 +34,7 @@ object BasicScenarios {
 
   val returnResourceAccessors: ScenarioBuilder = scenario("Return Resource Accessors")
     .forever(
-      feed(ReturnResourceAccessors.feed)
+      feed(CreateResourceAccess.feed)
         .exec(AccessManagement.createResourceAssess)
```

Comment:
i  am not sure why this change is required?

Reply:
This is just scala class setting default values for feed def feed: Feeder[String] = Iterator.continually {
    Map(
      "resourceId" -> randomUUID,
      "accessorId" -> randomUUID
    )
  }

we don't have any ReturnResourceAccessors feed.  and we can use the same.